guides/micronaut-jaxrs-jdbc/java/src/test/java/example/micronaut/PetResourceTest.java

```diff
@@ -0,0 +1,100 @@
+package example.micronaut;
+
+import io.micronaut.core.type.Argument;
+import io.micronaut.http.HttpRequest;
+import io.micronaut.http.HttpResponse;
+import io.micronaut.http.HttpStatus;
+import io.micronaut.http.client.HttpClient;
+import io.micronaut.http.client.annotation.Client;
+import io.micronaut.http.client.exceptions.HttpClientResponseException;
+import io.micronaut.http.uri.UriBuilder;
+import io.micronaut.test.extensions.junit5.annotation.MicronautTest;
+import org.junit.jupiter.api.Test;
+
+import jakarta.inject.Inject;
+import org.junit.jupiter.api.function.Executable;
+
+import java.net.URI;
+import java.util.Arrays;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+
+@MicronautTest(transactional = false) // <1>
```

Comment:
still don't know _why_ the test isn't transactional

Reply:
I'm planning to look into this more. This code was originally from the examples repo. I appreciate your thorough review, Bert and will be spending more time with this in the coming days.